shopinvader_price_per_qty/models/shopinvader_variant.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models
+
+
+class ShopinvaderVariant(models.Model):
+    _inherit = "shopinvader.variant"
+
+    def _get_qty_step(self):
+        items = self.env["product.pricelist.item"].search(
+            [
+                "|",
+                "|",
+                "|",
+                ("product_id", "=", self.record_id.id),
+                ("product_tmpl_id", "=", self.product_tmpl_id.id),
+                ("categ_id", "=", self.categ_id.id),
+                "&",
+                "&",
+                ("product_id", "=", False),
+                ("categ_id", "=", False),
+                ("product_tmpl_id", "=", False),
+            ]
+        )
+        return sorted({item.min_quantity for item in items if item.min_quantity > 1})
```

Comment:
@sebastienbeau you should take into account the date_start and date_end fields into your domain.